components/InteractiveElements/MessageInput.tsx

```diff
@@ -18,8 +19,24 @@ export default function MessageInput({
   const margin =
     useSelector((state: RootState) => state.screenHeight.height) * 0.01;
   const [messageFocus, setMessageFocus] = useState(false);
+  const [messageGraphemes, setMessageGraphemes] = useState(0);
+  const [splitter] = useState(new GraphemeSplitter());
   const messageLimit = 70;
```

Comment:
I like Fu's idea of moving to constants

Reply:
Will do!

Reply:
I'm going to address this in my other PR, and merge this one now